.github/MAINTAINER.adoc

```diff
@@ -0,0 +1,79 @@
+= Maintainers
+ifdef::env-github[]
+:status:
+:outfilesuffix: .adoc
+:caution-caption: :fire:
+:important-caption: :exclamation:
+:note-caption: :paperclip:
+:tip-caption: :bulb:
+:warning-caption: :warning:
+endif::[]
+
+== Labels
+
+bug/possible:: (only for issues) if we need to analyze to understand if it's a bug or not. [bot only]
+bug/confirmed:: (only for issues) we are sure, it's a bug.
+bug/fix:: (only for PR) it's a bug fix.
+
+contributor/need-more-information:: we need more information from the contributor in order to analyze a problem.
+contributor/waiting-for-corrections:: (only for PR) we need the contributor to take actions in order to move forward with a PR
+contributor/needs-rebase:: (only for PR) use it only when there is some conflicts (and an automatic rebase is not possible) [bot, humans]
+
+duplicate:: it's a duplicate issue/PR.
```

Comment:
Let's move `duplicate` next to `declined`.

I also suggest to prefix it with with `resolution/`.